src/anyon/qpu_interface.jl

```diff
@@ -32,6 +32,9 @@ struct MockRequestor<:Requestor
     post_checker::Function
 end
 
+get_request_throttle(::HTTPRequestor) = ()->sleep(0.1) # 100 ms between queries to the host
```

Comment:
I wouldn't say the request throttling is part of the requestor. To me, it's part of AnyonQPU. We are not throttling requests in general, only limiting how frequently we query the status.

Reply:
+1

I've moved it to `src/anyon/anyon.jl` as part of the `run_job`/`transpile_and_run_job` interface.

LMKWYT!